zap/src/test/java/org/parosproxy/paros/core/scanner/VariantMultipartFormParametersUnitTest.java

```diff
@@ -227,6 +229,305 @@ void shouldInjectParamValueModificationInFileNameParam() {
                 message.getRequestBody().toString(), equalTo(newMsg.getRequestBody().toString()));
     }
 
+    @Test
+    void shouldInjectSmallerParamValueModificationInFileNameParam() {
```

Comment:
Can be parameterized, it's just testing different values (same for some of the following tests).

Reply:
yeah

Reply:
I meant JUnit test parameterization (e.g. `VariantJSONQueryUnitTest`). The tests `shouldInjectParamValueMultipleTimesModificationsWith*` can also be parameterized.

Reply:
oh ok. I have done the changes as suggested. Please have a look.